doc/api/Renter.md

```diff
@@ -129,6 +129,18 @@ period // block height
 // fewer total transaction fees. Storage spending is not affected by the renew
 // window size.
 renewwindow // block height
+
+// Max download speed permitted, speed provide in bytes per second
+// setting is not persisted and will be reset by a shutdown
+maxdownloadspeed
+
+// Max upload speed permitted, speed provide in bytes per second
+// setting is not persisted and will be reset by a shutdown
+maxuploadspeed
+
+// Stream cache size specifies how many data chunks will be cached while 
+// streaming.  Must be greater than 0
+streamcachesize
```

Comment:
I don't think specifying `Must be greater than 0` is necessary; the same is true for many other settings, and the wording makes it sounds like you *have* to set `streamcachesize` explicitly.